src/ui/zabapgit_icon_font_css.w3mi.data.css

```diff
@@ -59,3 +59,7 @@
 .icon-snowflake:before { content: "\f11e"; }
 .icon-star:before { content: "\f11f"; }
 .icon-tools-solid:before { content: "\f120"; }
+
+.icon-cloud-upload-alt {

```

Comment:
This file is autogenerated (https://github.com/abapGit/icon-font), so better to fix it on common css.
I'd also try to unify it, it is strange to have global padding for one specific icon. Did you mean "icon in anchor"?
```css
a i.icon { padding-right: 5px }
```
just beware that it may have to reconcile with other icons near links (e.g. repo menu)

Reply:
What I was trying to do with this is that this icon was slightly wider, it looked like there was no space.

![](https://user-images.githubusercontent.com/5097067/86728649-eae21300-c02c-11ea-995f-2daa581917c5.png)

Reply:
Ah, I see. It makes sense then. then the style is OK, just move it to commons pls.